lib/scnnr/routing_utils.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Scnnr
+  class RoutingUtils
```

Comment:
Same as `PollingUtils`. Why do we rewrite the routing logic in this static way?

Reply:
I think you already treat me as criticizing this lib, I don't think any explanation won't work, please allow me to be silent.

That's my bad to not cautious about how to give a proposal in a friendly way. sorry if we talk face to face, this won't happen, because you can see my face and what mean.

But thank you for the dinner and the time spent one me.